reana_server/rest/launch.py

```diff
@@ -217,7 +217,13 @@ def launch(user, url, name="", parameters="{}", specification=None):
             jsonify({"message": "The workflow 'parameters' field is not valid JSON."}),
             400,
         )
-    except (REANAFetcherError, REANAValidationError, ValueError, ValidationError) as e:
+    except (
+        REANAFetcherError,
+        REANAValidationError,
+        REANAQuotaExceededError,
+        ValueError,
+        ValidationError,
+    ) as e:
         logging.error(traceback.format_exc())
         return jsonify({"message": str(e)}), 400
```

Comment:
*suggestion:* Here return code is 400. In `check_quota` it is 403. Let's have a separate catch for quota that returns HTTP 403.